app/experimenter/experiments/tests/test_bugzilla.py

```diff
@@ -44,6 +44,55 @@ def test_creating_pref_bugzilla_ticket_returns_ticket_id(self):
             },
         )
 
+    def test_create_bugzilla_ticket_retries_with_no_assignee(self):
+        experiment = ExperimentFactory.create_with_status(
+            Experiment.STATUS_DRAFT, name="An Experiment"
+        )
+
+        self.setUpMockBugzillaInvalidUser()
+
+        response_data = create_experiment_bug(experiment)
+
+        self.assertEqual(response_data, self.bugzilla_id)
+
+        self.assertEqual(self.mock_bugzilla_requests_post.call_count, 2)
```

Comment:
It's a minor thing, but the spacing in this function seems distracting.

Reply:
Oh yeah there was one too many newlines there.